GoogleForCreators/web-stories-wp
# Canvas: make mask elements focusable

## Context

Branched off of #11193.

Elements used to be selected when focused, but that was removed in #5968. This PR adds that back in.

Clicking on the frames around shapes should not select the elements behind the shape. That is consistent with other apps like canva and adobe illustrator. Ex: adding a heart shape will have some space between the shape and the frame. Clicking that space should not select any elements underneath the shape.

## Summary

Elements in the canvas are selected when focused.

## Relevant Technical Choices

n/a

## To-do

n/a

## User-facing changes

![tab](https://user-images.githubusercontent.com/22185279/162527164-adc7c6ed-4db9-4dcf-94d6-cc9ae19df66c.gif)

## Testing Instructions






This PR can be tested by following these steps:

1. Add one of every type of element to the canvas
2. Tab in the canvas until you reach the focus container
3. Press `Enter`
4. `Tab` and `Shift + Tab` around. Each time you focus a new element, it should be come 'selected' in the editor.


## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #11000
